### PR TITLE
Redesign monthly-plan page

### PR DESCRIPTION
**Pull request by flickz**
This PR redesigns the monthly plan page based on the figma design provided and adds the amount selection feature.

NOTE: I added a new package called [react-number-format](https://www.npmjs.com/package/react-number-format) basically to format the numbers to amount with currency.

#### Desktop

Screenshot 2019-09-10 at 9 02 50 AM

Screenshot 2019-09-10 at 9 12 28 AM

#### Mobile

Screenshot 2019-09-10 at 2 57 57 PM

Screenshot 2019-09-10 at 2 58 29 PM

TODO:

- [x] Pass the selected amount to OC contribution url

- [x] Fix some responsiveness glitches

### CODE REVIEW

**Comment by flickz** [comment]
Screenshot 2019-09-13 at 12 18 50 PM

Updated.

**Comment by znarf** [comment]
@flickz can you look at the design and double check the spacing above and below "employees"?

Currently, it seems in the middle while it should be closer to the range (ie: 1-10, 10-100).

File: src/pages/monthly-plan.js [hunk 1]
```diff
@@ -265,6 +265,8 @@ export default class MonthlyPlan extends React.Component {
                 font-size: 12px;
                 line-height: 18px;
                 color: #9D9FA3;
+                position: relative;
+                top: -5px;
```
Comment (znarf) [inline src/pages/monthly-plan.js]: Is that the best use of CSS? Why not use padding/marging?

**Comment by znarf** [comment]
@flickz I rebased and changed the target branch. Can you have a look and do more testing/polishing?

- [ ] the button should not be disabled by default

**Comment by flickz** [comment]
Sure

**Comment by flickz** [comment]
> we want the description of the collective next to the title, design from @raulgrafico is pending but we should already add it in a simple form

Screenshot 2019-09-19 at 10 19 19 AM

@znarf What do you think?

**Pull request merged**